package.json

```diff
@@ -1,10 +1,41 @@
 {
   "name": "@metamask/superstruct",
-  "description": "A simple and composable way to validate data in JavaScript (and TypeScript).",
   "version": "1.0.3",
-  "license": "MIT",
+  "description": "A simple and composable way to validate data in JavaScript (and TypeScript).",
+  "keywords": [
+    "api",
+    "array",
+    "assert",
+    "cast",
+    "check",
+    "checker",
+    "collection",
+    "data",
+    "error",
+    "express",
+    "hapi",
+    "interface",
+    "invalid",
+    "joi",
+    "json",
+    "list",
+    "model",
+    "object",
+    "orm",
+    "scalar",
+    "schema",
+    "struct",
+    "throw",
+    "type",
+    "types",
+    "valid",
+    "validate",
+    "validation",
+    "validator"
+  ],
   "repository": "git://github.com/MetaMask/superstruct.git",
-  "type": "module",
```

Comment:
I see that we're no longer using ESM. Is that intentional? Should we stick with ESM so it makes things easy?

Reply:
I'm planning to swap out Rollup for `tsup` using the same configs we use in the module template, which doesn't use `type: module`.